services/src/main/java/io/druid/cli/RouterJettyServerInitializer.java

```diff
@@ -50,7 +51,9 @@
 public class RouterJettyServerInitializer implements JettyServerInitializer
 {
   private static List<String> UNSECURED_PATHS = Lists.newArrayList(
-      "/status/health"
+      "/status/health",
+      // JDBC authentication uses the JDBC connection context instead of HTTP headers, skip the normal auth checks
```

Comment:
Please include a comment about how the context will be forwarded to brokers and they will verify it.

Reply:
Added a comment on the broker being responsible for the auth checks